release-notes/6.0.0.md

```diff
@@ -135,5 +134,59 @@ In this release, we have added automation script to create the reports using rep
 
     Please note that the specific steps may vary depending on your Jenkins configuration and environment. Ensure you have the necessary permissions and access rights to perform these actions.
 
-    
+    report_id's to run in the RunReportjob:
```

Comment:
@Sachin its not clear please provide more details

Reply:
@rakeshSgr added some more details please review them once.